<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSave</class>
 <widget class="QDialog" name="FormSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SALARA PROJECT   |   Space Object Application</string>
  </property>
  <property name="accessibleName">
   <string>form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDateTimeEdit
{ 
    font: 9pt &quot;Open Sans&quot;;
    color: rgb(253,245,230);
    background-color: rgb(70, 70,70);
}
QDateTimeEdit:disabled
{
	color: rgb(100, 100,100);
}

/*--------------------------------*/
/* QPlainTextEdit */
/*--------------------------------*/
QPlainTextEdit
{
    color: rgb(253,245,230);
    background-color: rgb(70, 70,70);
    padding: 2px 2px;
    font: 9pt &quot;Open Sans&quot;;
}
QPlainTextEdit:disabled
{
    color: rgb(100, 100,100);
}
QPlainTextEdit:focus 
{
	border: 1px solid rgb(255,155,0);
}

QPushButton
{
     color: rgb(215, 215, 215);
     background-color: rgb(53,53,53);
	 font:  9pt &quot;Open Sans Semibold&quot;;
}

QPushButton:disabled
{
	color: rgb(100, 100,100);
}

QPushButton[accessibleName=&quot;Blue&quot;]
{
	background-color:  rgb(20,20,100);
}

QPushButton[accessibleName=&quot;Lilac&quot;]
{
	border-style: solid;
	border-width: 1px;
	border-color: black;
	border-radius: 5px;
	background-color:  rgb(50,50,100);
}
QPushButton[accessibleName=&quot;Lilac&quot;]:hover
{
	background-color:  rgb(50,60,100);
}

QPushButton[accessibleName=&quot;Green&quot;]
{
	border-style: solid;
	border-width: 1px;
	border-color: black;
	border-radius: 5px;
	background-color: rgb(50, 90, 0);
}

QPushButton[accessibleName=&quot;Green&quot;]:hover
{
	background-color: rgb(50, 100, 0);
}

QPushButton[accessibleName=&quot;Red&quot;]
{
	border-style: solid;
	border-width: 1px;
	border-color: black;
	border-radius: 5px;
    background-color:  rgb(140, 20, 0);
}

QPushButton[accessibleName=&quot;Red&quot;]:hover
{
     background-color:  rgb(150, 20, 0);
}




/*--------------------------------*/
/* Widget forms */
/*--------------------------------*/
QWidget[accessibleName=&quot;form&quot;]
{
	background-color: rgb(53, 53, 53);
}
/*--------------------------------*/

/*--------------------------------*/
/* QLineEdit */
/*--------------------------------*/
QLineEdit
{
    color: rgb(253,245,230);
    background-color: rgb(70, 70,70);
    padding: 2px 2px;
    font: 9pt &quot;Open Sans&quot;;
}
QLineEdit:disabled
{
    color: rgb(100, 100,100);
}
QLineEdit[error=true]
{
    background-color: rgb(180,8,0);
}
QLineEdit[error=true]:focus 
{
	border: 1px solid rgb(255,155,0);
}
QLineEdit:focus 
{
	border: 1px solid rgb(255,155,0);
}
/*--------------------------------*/

/*--------------------------------*/
/* QSpinBox */
/*--------------------------------*/
QSpinBox
{ 
    font: 8pt &quot;Open Sans&quot;;
    color: rgb(253,245,230);
    background-color: rgb(70, 70,70);
    padding: 2px 2px;
}
QSpinBox:disabled
{
    color: rgb(100, 100,100);
}
/*--------------------------------*/

/*--------------------------------*/
/* QDoubleSpinBox */
/*--------------------------------*/
QDoubleSpinBox
{ 
    font: 8pt &quot;Open Sans&quot;;
    color: rgb(253,245,230);
    background-color: rgb(70, 70,70);
    padding: 2px 2px;
}
QDoubleSpinBox:disabled
{
	color: rgb(100, 100,100);
}
/*--------------------------------*/

/*--------------------------------*/
/* QLabel */
/*--------------------------------*/
QLabel
{
    color: rgb(253,245,230);
	font: 9pt &quot;Open Sans Semibold&quot;;
}
QLabel:disabled
{
    color: rgb(100, 100,100);
}
/*--------------------------------*/

/*--------------------------------*/
/* QGroupBox */
/*--------------------------------*/
QGroupBox
 {
	font: 9pt &quot;Open Sans Extrabold&quot;;
	color: rgb(253,245,230);
    border: 3px solid gray;
    border-radius: 10px;
    margin-top: 0.6em;
    padding: 3px;
}
QGroupBox:disabled
{
    color: rgb(100, 100,100);
}
/*--------------------------------*/

/*--------------------------------*/
/* QComboBox */
/*--------------------------------*/
QComboBox
{
    font: 8pt &quot;Open Sans&quot;;
    color: rgb(253,245,230);
    padding: 2px 8px 2px 8px;
    selection-background-color: rgb(255,155,0);
	background-color:  rgb(70, 70,70);
}
QComboBox::item
{
    font: 8pt &quot;Open Sans&quot;;
    color: rgb(253,245,230);
	background-color:  rgb(70, 70,70);
}
QComboBox::item:selected
{
	background-color:  rgb(255,155,0);
}
QComboBox::indicator
{
    background-color:transparent;
    selection-background-color:transparent;
    color:transparent;
    selection-color:transparent;
}
/*--------------------------------*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Open Sans Semibold</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Before continuing, check the complete file path and the version name (you can modify both). Also you can write a comment if you consider it necessary (you can include some relevant change in the data, your name, etc). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Open Sans Semibold</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Complete path:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="le_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_file">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../SP_Core/resources/common_resources.qrc">
           <normaloff>:/icons/link_icon&amp;48.png</normaloff>:/icons/link_icon&amp;48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Open Sans Semibold</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Version name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_versionname">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Open Sans Semibold</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Version date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dte_date">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>0</height>
          </size>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="minimumDate">
          <date>
           <year>2018</year>
           <month>9</month>
           <day>13</day>
          </date>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::DaySection</enum>
         </property>
         <property name="displayFormat">
          <string>dd-MM-yyyy   hh:mm:ss</string>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Open Sans Semibold</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>File comment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="te_comment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1646446</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="accessibleName">
        <string>Green</string>
       </property>
       <property name="text">
        <string> Save Data</string>
       </property>
       <property name="icon">
        <iconset resource="../SP_Core/resources/common_resources.qrc">
         <normaloff>:/icons/save_icon&amp;48.png</normaloff>:/icons/save_icon&amp;48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="accessibleName">
        <string>Red</string>
       </property>
       <property name="text">
        <string> Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../SP_Core/resources/common_resources.qrc">
         <normaloff>:/icons/cancel_icon&amp;48.png</normaloff>:/icons/cancel_icon&amp;48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../SP_Core/resources/common_resources.qrc"/>
  <include location="../SP_Core/resources/common_resources.qrc"/>
 </resources>
 <connections/>
</ui>
